### PR TITLE
xss test for per-host recent jobs popup

### PR DESCRIPTION
**Pull request by chrismeyersfsu**
related to https://github.com/ansible/ansible-tower/issues/7855

### CODE REVIEW

**Review by jakemcdermott — APPROVED** [review]

File: awx/ui/test/e2e/tests/test-xss.js [hunk 1]
```diff
@@ -54,13 +59,15 @@ module.exports = {
...
                 urls.inventory = `${pages.inventories.url()}/inventory/${data.inventory.id}`;
+                urls.hosts = `${pages.hosts.url()}`;
                 urls.inventoryScript = `${pages.inventoryScripts.url()}/${data.inventoryScript.id}`;
                 urls.inventorySource = `${urls.inventory}/inventory_sources/edit/${data.inventorySource.id}`;
                 urls.sourceSchedule = `${urls.inventorySource}/schedules/${data.sourceSchedule.id}`;
                 urls.smartInventory = `${pages.inventories.url()}/smart/${data.smartInventory.id}`;
                 urls.project = `${pages.projects.url()}/${data.project.id}`;
                 urls.credential = `${pages.credentials.url()}/${data.credential.id}`;
                 urls.jobTemplate = `${pages.templates.url()}/job_template/${data.jobTemplate.id}`;
+                urls.jobTemplates = `${pages.templates.url()}`;
```
Comment (jakemcdermott) [inline awx/ui/test/e2e/tests/test-xss.js]: I'm not seeing where this additional url (urls.jobTemplates) gets used in the test - is this used somewhere?

File: awx/ui/test/e2e/fixtures.js [hunk 2]
```diff
@@ -182,7 +195,7 @@ const waitForJob = endpoint => {
                 const completed = statuses.indexOf(update.data.status) > -1;
 
                 if (completed) {
-                    return resolve();
+                    return resolve(update.data);
```
Comment (jakemcdermott) [inline awx/ui/test/e2e/fixtures.js:198]: 👍

File: awx/ui/test/e2e/tests/test-xss.js (hunk 1 above)
Comment (chrismeyersfsu) [reply awx/ui/test/e2e/tests/test-xss.js]: I think this was a stream crossing. The other xss, I thought needed the job templates tab since the schedules button/tab is usually gotten to by first visiting the job templates page. However, the UI is robust enough to go directly to that tab.

TL;DR will remove
